doc/faq.mkd

```diff
@@ -46,7 +46,23 @@ on your remote repositories:
 [git-clone]: https://www.kernel.org/pub/software/scm/git/docs/git-clone.html "git-clone(1)"
 [git-symbolic-ref]: https://www.kernel.org/pub/software/scm/git/docs/git-symbolic-ref.html "git-symbolic-ref(1)"
 
-[github-default-branch]: https://help.github.com/articles/setting-the-default-branch "Setting the default branch"
-[bitbucket-default-branch]: https://answers.atlassian.com/questions/280944/how-to-change-main-branch-in-bitbucket "How to change MAIN branch in BitBucket?"
-[gitolite-v2-default-branch]: http://stackoverflow.com/questions/7091599/git-default-remote-branch-with-gitolite "git default remote branch with gitolite"
-[gitolite-v3-default-branch]: http://stackoverflow.com/questions/13949093/git-change-default-branch-gitolite "git change default branch (gitolite)"
+[github-default-branch]: https://help.github.com/articles/setting-the-default-branch
```

Comment:
> That being said I don't think this is worth changing.

I disagree.  It's unnecessary to set the title text here, as the link text is sufficient.  More importantly, most people who use screen readers would hear [both the link text and the title text](http://www.deque.com/blog/text-links-practices-screen-readers/) (see the first row of the table under The Comparison), potentially confusing them.  As such, I think it makes sense to remove it.  You are right that this should be done in a separate commit, though.